overlay/osx/qt5-base/portfile.cmake

```diff
@@ -173,6 +173,12 @@ elseif(VCPKG_TARGET_IS_OSX)
             "QMAKE_MACOSX_DEPLOYMENT_TARGET = 10.12" 
             "QMAKE_MACOSX_DEPLOYMENT_TARGET = ${VCPKG_OSX_DEPLOYMENT_TARGET}" 
             _tmp_contents "${_tmp_contents}")
+        if("${VCPKG_TARGET_ARCHITECTURE}" MATCHES "arm64")
+            string(REPLACE 
+                "QMAKE_APPLE_DEVICE_ARCHS = x86_64" 
+                "QMAKE_APPLE_DEVICE_ARCHS = arm64" 
+                _tmp_contents ${_tmp_contents})
+        endif()
```

Comment:
can you explain this? Is this a change from us or a patch from Qt5.15?

Reply:
Ups, this slips in from the M1 arm port.
I must have messed up the rebase. 
I will rebase this again to remove that change.